modules/mobile/app/services/mobile/v0/legacy_payment_information/proxy.rb

```diff
@@ -2,7 +2,7 @@
 
 module Mobile
   module V0
-    module PaymentInformation
+    module LegacyPaymentInformation
```

Comment:
I could also have renamed this EVSS instead of legacy. But I think Legacy makes it clearer that this shouldn't be around long.